package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "cdmx009-mdlinks",
+  "version": "1.0.0",
+  "description": "CDMX009 MDLinks Mariana Olimpia",
```

La descripción tiene que ser sobre el uso de la librería ademas tendría que estar en ingles
```suggestion
  "description": "Library for read md files for validate and stats",
```